src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4; // 16
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+    private Node<K, V> nullKeyNode;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
The usage of raw types in generics is not safe. Use the diamond operator or specify actual type arguments to ensure type safety. For instance, it should be 'new Node<>[DEFAULT_INITIAL_CAPACITY]' or 'new Node<K,V>[DEFAULT_INITIAL_CAPACITY]'.